utam-compiler/src/main/java/utam/compiler/translator/Manifest.java

```diff
@@ -0,0 +1,76 @@
+package utam.compiler.translator;
+
+import static utam.core.framework.UtamLogger.info;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.util.DefaultIndenter;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/** Writes manifest of generated files */
+public class Manifest {
```

Comment:
nit: javadoc format seems off (needs author and since)

Reply:
Didn't realize we were making that mandatory for class JavaDoc. Fixed in subsequent commit.